lib/Doctrine/DBAL/Migrations/Tools/Console/Command/AbstractCommand.php

```diff
@@ -117,20 +117,13 @@ protected function getMigrationConfiguration(InputInterface $input, OutputInterf
     }
 
     /**
-     * This method ensure that we stay compatible with symfony console 2.3 by using the deprecated dialog helper
-     * but use the ConfirmationQuestion when available.
-     *
-     * @param $question
+     * @param string $question
      * @param InputInterface $input
      * @param OutputInterface $output
      * @return mixed
      */
-    protected function askConfirmation($question, InputInterface $input, OutputInterface $output)
+    protected function askConfirmation(string $question, InputInterface $input, OutputInterface $output)
```

Comment:
Technically, this is a BC break - people might have overridden this method to add their own logic.